lib/ui/notifications.dart

```diff
@@ -62,48 +62,117 @@ class NotificationsWidgetState extends AssistantState<NotificationsWidget> {
     return DefaultTabController(
       length: 2,
       child: Scaffold(
-        floatingActionButton: Visibility(
-          visible: showHomeButton,
```

Comment:
Maybe just a bad merge, but your change removes the home button from the bottom nav